clicommand/agent_start.go

```diff
@@ -1182,7 +1182,16 @@ var AgentStartCommand = cli.Command{
 			pool.StartStatusServer(ctx, l, cfg.HealthCheckAddr)
 		}
 
-		return pool.Start(ctx)
+		err = pool.Start(ctx)
+		if errors.Is(err, agent.ErrJobAcquisitionFailure) {
+			// If the agent tried to acquire a job, but it couldn't because the job was already taken, we should exit with a
+			// specific exit code so that the caller can know that this job can't be acquired.
+
+			const acquisitionFailedExitCode = 27 // chosen by fair dice roll
```

Comment:
nope, AFAICT, we only ever return 0 or 1. the reason i didn't go with `2` is that it implies that this is the second most important exit code. because it's one of two non-zero exit codes, it is the second-most important exit code by definition, but were we to add a third, this would likely not be the case.

regardless, potentially it's a good idea to hoist this const definition out from this function and put it somewhere central-ish in the `clicommand` package?

Reply:
Sounds good to me